### PR TITLE
chore(global): reorganize monorepo

### PR DESCRIPTION
**Pull request by jogold**
Add dev dependencies, .gitignore, lint and test scripts in each package

### CODE REVIEW

**Comment by jogold** [comment]
@belgac @dalawwa review reminder

File: package.json [hunk 1]
```diff
@@ -16,63 +16,40 @@
...
-    "test": "yarn lint && yarn type-check && yarn jest",
-    "type-check": "tsc --noEmit"
+    "sort": "lerna run --stream sort",
+    "test": "lerna run --stream test",
+    "type-check": "lerna run --stream type-check"
+  },
+  "eslintConfig": {
+    "extends": "@goldwasserexchange"
   },
   "husky": {
```
[the remark this answers is not in the document]
Comment (jogold) [reply package.json]: https://github.com/typicode/husky/issues/108

Only problematic for `pre-push`, `post-checkout`, `post-commit` and `post-merge`? https://github.com/typicode/husky/issues/108#issuecomment-436133463

[the remark this answers is not in the document]
Comment (belgac) [reply package.json]: I think so yes

[the remark this answers is not in the document]
Comment (jogold) [reply package.json]: OK, can you approve?

**Pull request merged**